GitUI/CommandsDialogs/FormResolveConflicts.cs

```diff
@@ -788,25 +788,14 @@ private void HandleSingleSelect()
             string remoteSide = GetRemoteSideString();
             string localSide = GetLocalSideString();
 
-            if (baseFileExists && localFileExists && remoteFileExists)
+            conflictDescription.Text = (baseFileExists, localFileExists, remoteFileExists) switch
             {
-                conflictDescription.Text = string.Format(_fileChangeLocallyAndRemotely.Text, localSide, remoteSide);
-            }
-
-            if (!baseFileExists && localFileExists && remoteFileExists)
-            {
-                conflictDescription.Text = string.Format(_fileCreatedLocallyAndRemotely.Text, localSide, remoteSide);
-            }
-
-            if (baseFileExists && !localFileExists && remoteFileExists)
-            {
-                conflictDescription.Text = string.Format(_fileDeletedLocallyAndModifiedRemotely.Text, localSide, remoteSide);
-            }
-
-            if (baseFileExists && localFileExists && !remoteFileExists)
-            {
-                conflictDescription.Text = string.Format(_fileModifiedLocallyAndDeletedRemotely.Text, localSide, remoteSide);
-            }
+                (true, true, true) => string.Format(_fileChangeLocallyAndRemotely.Text, localSide, remoteSide),
+                (false, true, true) => string.Format(_fileCreatedLocallyAndRemotely.Text, localSide, remoteSide),
+                (true, false, true) => string.Format(_fileDeletedLocallyAndModifiedRemotely.Text, localSide, remoteSide),
+                (true, true, false) => string.Format(_fileModifiedLocallyAndDeletedRemotely.Text, localSide, remoteSide),
+                _ => null
```

Comment:
This overwrites the the default text
```suggestion
                _ => conflictDescription.Text
```